src/components/countdown/countdown.jsx

```diff
@@ -42,6 +42,9 @@ module.exports = React.createClass({
   },
 
   onCountdown: function(time) {
+    if (this.state.countdownManager.hasDatePassed()) {
```

Comment:
looks like this should be props, not state. problem not introduced here but could be fixed... should countdownManager be part of the state?